### PR TITLE
Review Emi

### PR DESCRIPTION
**Pull request by emideluxe**
These are my changes. @wires can you have a look as well, please.

### CODE REVIEW

File: css/theme/marcosh.css [hunk 1]
```diff
@@ -56,8 +56,8 @@ pre code {
 .reveal footer img {
   position: fixed;
   bottom: 10px;
-  height: 70px;
-  width: 80px;
+  height: 140px;
```
Comment (marcosh) [inline css/theme/marcosh.css:59]: making it this big interferes with the slides, since it overlaps with them

File: fun-with-categories.md [hunk 2]
```diff
@@ -1,32 +1,36 @@
-# 🎉 FUN WITH CATEGORIES 🎉
 
----
+## 🎉 FUN WITH CATEGORIES 🎉
 
-## Marco Perone
+#### Marco Perone
 
-[twitter.com/marcoshuttle](https://twitter.com/marcoshuttle)
+![statebox](img/statebox_simpleblack_txt_bottom.png)
```
Comment (marcosh) [inline fun-with-categories.md]: I think that the talk should be focused on its topic, which is not Statebox itself, and in this case I'd prefer to reserve a slide for Statebox

File: fun-with-categories.md [hunk 3]
```diff
@@ -1,32 +1,36 @@
...
 
-[marcosh.github.com](https://marcosh.github.com)
+<!-- TODO add better layout for the title page -->
 
-[medium.com/@marcosh](https://medium.com/@marcosh)
+---
+
+## About Marco
+
+- PhD Mathematics, *University of Padua*
```
Comment (marcosh) [inline fun-with-categories.md:14]: I don't want to mention any detail about me, I'll just say I'll work at Stetebox

File: fun-with-categories.md [hunk 4]
```diff
@@ -53,7 +57,7 @@ what is category theory?
 
 even among mathematicians sometimes described as abstract nonsense
 
-but that is good beacuse:
+but that is good because:
```
Comment (marcosh) [inline fun-with-categories.md:58]: thanks a lot for correcting the typos! Almost all of them are in the notes which are never shown, but still it's better to have them correct

File: fun-with-categories.md [hunk 5]
```diff
@@ -125,7 +129,7 @@ concurrent and distibuted computations
 
 ---
 
-[github.com/statebox/awesome-applied-category-theory](https://github.com/statebox/awesome-applied-category-theory)
+[github.com/statebox/awesome-applied-ct](https://github.com/statebox/awesome-applied-category-theory)
```
Comment (marcosh) [inline fun-with-categories.md:130]: could I ask why did you change the name of the repo? The original name was longer but definitely unambiguous and clearer to grasp for people which don't do ct for a job

File: fun-with-categories.md [hunk 6]
```diff
@@ -465,24 +469,29 @@ Provide initial value
 
 ---
 
-![solong](img/solong.jpg)
-
----
+![categorify all the things!](img/categorify-all-the-things.png)
```
Comment (marcosh) [inline fun-with-categories.md:470]: I must admit that I hate this kind of memes, so I'd really like to stick with my original image. Also, an HGTTG is always nice

File: index.html [hunk 7]
```diff
@@ -32,11 +32,11 @@
 			<footer>
                 <div class="details">
                 Lambda World - Cadiz, October 17<sup>th</sup>/18<sup>th</sup> 2019 -
-                <a href="https://twitter.com/marcoshuttle">@marcoshuttle</a>
+                <a href="https://twitter.com/statebox">@statebox</a>
```
Comment (marcosh) [inline index.html:35]: I think that attendees of the conference who want to interact with the speaker should have the speaker handle easily available, not his company one. Anyway, the Statebox twitter account is mentioned already in the slides

File: index.html [hunk 8]
```diff
@@ -8,7 +8,7 @@
 
 		<link rel="stylesheet" href="css/reset.css">
 		<link rel="stylesheet" href="css/reveal.css">
-		<link rel="stylesheet" href="css/theme/solarized.css">
+		<link rel="stylesheet" href="css/theme/white.css">
```
Comment (marcosh) [inline index.html:11]: I really prefer the font and the background colour of the theme I chose, so I'd rather stick to that

**Review by marcosh — COMMENTED** [review]
Hi Emi, thanks for the feedback

I already included some of your comments and corrections.

For some other, I am quite reluctant to include them. I tried to explain my reasoning in the comments below

File: css/theme/marcosh.css (hunk 1 above)
Comment (emideluxe) [reply css/theme/marcosh.css:59]: This depends on the screen size/resolution. I scaled it by 2, maybe scale it by 1.5?

File: fun-with-categories.md (hunk 2 above)
Comment (emideluxe) [reply fun-with-categories.md]: Well, Statebox is about applying category theory and we present our category theory library here. You do the presenting because you worked a lot on this library, but it was a collaborative Statebox effort. At this stage we really want to take every opportunity to promote the project and build our community, this is the main reason why we apply for talks.

File: fun-with-categories.md (hunk 3 above)
Comment (emideluxe) [reply fun-with-categories.md:14]: I think it's interesting to the audience to mention your background in ring theory and that you are a PHP programmer, listeners might relate to this and want to talk to you about it. But of course this is up to you.

File: fun-with-categories.md (hunk 4 above)
Comment (emideluxe) [reply fun-with-categories.md:58]: Yes, I was aware of this, but in case someone looks at the slides and the notes it's better to have it free of spelling errors.

File: fun-with-categories.md (hunk 5 above)
Comment (emideluxe) [reply fun-with-categories.md:130]: The longer name did not fit on the slides and this seemed like a good compromise. The longer name is still available as a redirect of course.

File: fun-with-categories.md (hunk 6 above)
Comment (emideluxe) [reply fun-with-categories.md:470]: Hitchhiker is great of course, but irrelevant to the talk. The take home message of this presentation should be that Statebox is building the tools to apply category theory and the last slides is where you reiterate the message.

File: index.html (hunk 7 above)
Comment (emideluxe) [reply index.html:35]: In my experience those that want to talk to you will come up to you after the presentation. Furthermore the last slide is usually kept visible as you answer questions, this is when people start following on twitter.

We could put both `@statebox` and `@marcosh` in the footer if you want?

File: index.html (hunk 8 above)
Comment (emideluxe) [reply index.html:11]: I'm afraid this is not a personal preference matter. As I previously mentioned, you are representing Statebox.

Looking forward to see the changes and good luck with the talk!
